app/logger_config.py

```diff
@@ -12,15 +14,68 @@ def format(self, record):
         else:
             record.url = None
             record.remote_addr = None
-
         return super().format(record)
 
 
+class CloudWatchHandler(logging.Handler):
```

Comment:
Have you deployed this to test-one to test? From reading some comments, around SO suggesting this, it seems uncertain whether it is recommended to do this /whether it would work:
https://stackoverflow.com/questions/46101977/redirect-aws-lambda-logs-to-a-particular-log-group-in-cloudwatch/77214307#77214307
https://stackoverflow.com/questions/68544678/single-lambda-multiple-cloudwatch-log-groups

Reply:
Did see those stackoverflow links earlier, I did want to test this on test one to confirm as it looks like we may have a situation where we have the created log groups and the default lambda log group (the first link)